programmers/난이도별/level01.가운데_글자_가져오기/rockmiin.py

```diff
@@ -0,0 +1,5 @@
+def solution(s):
+    if(len(s)%2==0):
+        return s[len(s)//2-1:len(s)//2+1]
+    else:
+        return s[len(s)//2]
```

Comment:
len(s)가 지속적으로 사용되고 있습니다. 이럴 때 변수로 저장해주세요! 안 그러면 매번 배열의 갯수를 계산해야 합니다.
```suggestion
    sLen = len(s), half = sLen // 2
    if(sLen % 2 == 0):
        return s[half - 1 : half + 1]
    else:
        return s[half]
```

그리고 위의 코드는 다음과 같이 표현할 수 있습니다.

```python
    sLen = len(s), half = sLen // 2
    return s[half - 1 : half + 1] if sLen % 2 == 0 else s[half]
```